amundsen_application/static/js/components/common/Alert/index.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+
+import AlertIcon from './AlertIcon';
+
+import './styles.scss';
+
+export interface AlertProps {
+  message: string | React.ReactNode;
+  actionText?: string;
+  onAction?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+const Alert: React.FC<AlertProps> = ({
+  message,
+  onAction,
+  actionText,
+}: AlertProps) => {
+  return (
+    <div className="alert">
+      <AlertIcon />
+      <p className="alert-message">{message}</p>
+      {actionText && onAction && (
+        <button type="button" className="btn btn-link" onClick={onAction}>
```

Comment:
For our internal use case, the text will end up being a link to another Amundsen page. Given our desire to use the correct components for specific actions, do you think we should still support a button with onClick vs and anchor with an href...or both?

Reply:
Oh, I see.
I could make it support both easily. I will add an optional actionHref prop, if that has a value, I will render a router Link instead